src/server/route.rs

```diff
@@ -12,11 +12,16 @@ use crate::{router::Router, Endpoint};
 pub struct Route<'a, State> {
     router: &'a mut Router<State>,
     path: String,
+    prefix: bool,
```

Comment:
We could probably use a doc comment here to help share what this is for.

Reply:
Done!